volatility3/framework/plugins/windows/filescan.py

```diff
@@ -60,6 +66,13 @@ def scan_files(
     def _generator(self):
         kernel = self.context.modules[self.config["kernel"]]
 
+        if self.config["physical"]:
+            physical_layer = self.context.layers[kernel.layer_name]
```

Comment:
I doubt the kernel layer would ever be a physical layer?  I don't think this will ever get triggered properly?  Layer on you also try `physical_layer.mapping` which, if it's a DataLayer, won't work.  I think maybe the naming is off here?